worth2/templates/main/dashboard.html

```diff
@@ -10,7 +10,20 @@
             <h1>E-WORTH</h1>
             <div class="subhead">Empowering African-American Women on the Road to Health</div>
             <p class="lead">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas eu urna sit amet sapien dapibus egestas et eget nibh. Nulla feugiat bibendum enim eget pharetra. Sed id porttitor diam. Nam id mi ullamcorper, egestas neque sit amet, consequat metus. Donec sollicitudin metus quis consectetur malesuada. Nullam eget arcu dictum turpis imperdiet malesuada eget in nibh. Sed tellus diam, malesuada eget nisi a, pharetra aliquam metus. Integer finibus, lectus in sagittis maximus, erat orci egestas odio, nec interdum magna tortor eget odio. In sit amet suscipit sem. Aliquam tempor orci ut tincidunt semper. In pharetra diam in quam condimentum vehicula. Praesent aliquet mi vel suscipit placerat. Curabitur eget elementum enim.
+            Welcome to E-WORTH! As part of this course you will find information and resources to help
+            you protect your own health, the health of your family, and of your community. You can move
+            through the sections at your own pace and you don’t have to complete it all at once – you can
+            stop and come back. While we recommend that you move through the
+            sections in order you can skip ahead.
+            </p>
+            <p class="lead">
+            We do ask that you to complete E-WORTH <i>within 90 days</i>. Accounts and associated data
```

Comment:
minor grammar fix here - "We do ask that you to complete E-WORTH..." is a little awkward. Could probably be "We do ask that you complete E-WORTH...".

Also, it's better practice to use `<em>` tags over `<i>` tags.

Reply:
great - will fix. and thx for the reminder re: em vs i